docs/redoc/master/openapi.json

```diff
@@ -3264,6 +3264,7 @@
         "type": "object",
         "required": [
           "commit",
+          "is_voter",
```

Comment:
if the field is required, we would need to update the client as well. Otherwise API calls will fail

Reply:
It won't be a minor release anymore

Reply:
This is a return type for the cluster status. I think by default any Json deserializer will ignore extra fields?

Reply:
well, not really. We could disable the strict check, though

Reply:
ok, i checked it, it is fine